traffic_info_client.py

```diff
@@ -0,0 +1,72 @@
+"""
+TCP-client collect the information about internet traffic on PC
+every 60 seconds and send in on the server.
+
+For cancel the program running press Ctrl+Q.
+"""
+import keyboard
+import psutil
+import socket
+import time
+
+
+sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+sock.connect(('localhost', 9090))
+
+running = False
+
+
+def traffic_info(interval=60):
+    """
+    Collect the information about internet traffic.
+
+    :param interval: interval (in seconds) of sending
+                     information to the server
+    """
+
+    global running
+    running = True
+    start_time = time.time()
+    while running:
+        prev_recv_bytes = psutil.net_io_counters(pernic=False).bytes_recv
+        start_sent_bytes = psutil.net_io_counters(pernic=False).bytes_sent
+
+        while time.time() - start_time < interval and running:
+            # Здесь можно собирать, например, температуру,
+            # суммировать значения и считать количество замеров
+            time.sleep(1)
+        if not running:
+            sock.close()
+            break
+
+        # А здесь посчитать среднее арифметичесое температуры за 1 минуту
+        recv_bytes = (psutil.net_io_counters(pernic=False).bytes_recv
+                            - prev_recv_bytes)
```

Comment:
Please check PEP8 for these line: continuation line over-indented for visual indent

Reply:
Fixed